configure

```diff
@@ -196,14 +196,14 @@ for tool in sed ocaml ocamlc uname rm make cat dirname basename; do
     fi
 done
 
-lib_suffix=`ocamlc -config 2>/dev/null|tr -d '\015'|sed -n -e 's/^ext_lib: //p'`
+lib_suffix=$(ocamlc -config 2>/dev/null | tr -d '\015' | sed -n -e 's/^ext_lib: //p')
```

Comment:
We could POSIX-ly clearer here and have:
```suggestion
lib_suffix=$(ocamlc -config 2>/dev/null | tr -d '\r' | sed -n -e 's/^ext_lib: //p')
```
(and in other places)

Reply:
Thanks, I applied this to all occurences of `\015`!